docs/Installation.md

```diff
@@ -65,6 +65,10 @@ sudo cpan -i Zonemaster::Backend
 Check the [declaration of prerequisites] to make sure your preferred combination
 of operating system version and database engine version is supported.
 
+Add Zonemaster user:
+sudo useradd zonemaster

```

Comment:
Github keeps collapsing my review comments so I'm pasting the conversation here.

@mattias-p said:
> The `useradd` command should probably be invoked with options `-r -c "Zonemaster daemon user"` to make it a system account with a description. Please check on CentOS the user created gets a `nologin` shell, and if not add `-s /usr/sbin/nologin` (possibly adjusting the path to `nologin`).

@sandoche2k replied:
> Why do we need a loginshell for the zonemaster user here?

My point is that we don't. That's why it should have the `nologin` default shell.